Buildings/Fluid/HeatExchangers/BaseClasses/VariableSpeedThermalWheels/BaseClasses/Characteristics/motorEfficiencyParameters.mo

```diff
@@ -0,0 +1,26 @@
+within Buildings.Fluid.HeatExchangers.BaseClasses.VariableSpeedThermalWheels.BaseClasses.Characteristics;
+record motorEfficiencyParameters
+  "Record for motor efficiency parameters vs. wheel speed ratio"
```

Comment:
`Parameters for defining motor efficiency at different wheel speed ratio`

Reply:
Update the model doc as suggested.